### PR TITLE
Added boot counting to unified kernels if /etc/kernel/tries exists

### PR DESCRIPTION
**Pull request by mcassaniti**
See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/ for details on boot counting

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
                 image_id = f"mkosi-{args.distribution}"
                 partlabel = None
 
+            # See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
+            boot_count: str = ""
```
Comment (behrmann) [inline mkosi/__init__.py:4058]:
```suggestion
            boot_count = ""
```
Currently travelling, so I can't check this right away, but mypy should figure this type out by itself. Explicitly adding types is usually only necessary, when annotating empty containers.

**Review by behrmann — COMMENTED** [review]
LGTM. Nice addition! Just a minor nit.

File: mkosi/__init__.py (hunk 1 above)
Comment (mcassaniti) [reply mkosi/__init__.py]: Done

File: mkosi/__init__.py [hunk 2]
```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
                 image_id = f"mkosi-{args.distribution}"
                 partlabel = None
 
+            # See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
```
Comment (keszybz) [inline mkosi/__init__.py:4057]: https://systemd.io/BOOT_LOADER_SPECIFICATION/#boot-counting is a better link.

File: mkosi/__init__.py [hunk 3]
```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
...
+            boot_count = ""
+            if root.joinpath("etc/kernel/tries").exists():
+                boot_count=f'+{root.joinpath("etc/kernel/tries").read_text().strip()}'
```
Comment (keszybz) [inline mkosi/__init__.py:4060]: Whitespace is missing around "=".

Please write this as:
```python
boot_count = f'…' if root.joinpath("etc/kernel/tries").exists() else ''
```

**Review by keszybz — CHANGES_REQUESTED** [review]
Looks reasonable.

File: mkosi/__init__.py (hunk 2 above)
Comment (mcassaniti) [reply mkosi/__init__.py:4057]: Done

File: mkosi/__init__.py (hunk 3 above)
Comment (mcassaniti) [reply mkosi/__init__.py:4060]: Done

**Comment by mcassaniti** [comment]
Looks like a random CI failure but everything else looks fine.

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**